tests/lib/nested.sh

```diff
@@ -471,6 +519,7 @@ start_nested_core_vm(){
     PARAM_NETWORK="-net nic,model=virtio -net user,hostfwd=tcp::$SSH_PORT-:22"
     PARAM_MONITOR="-monitor tcp:127.0.0.1:$MON_PORT,server,nowait"
     PARAM_USB="-usb"
+    PARAM_CD="${PARAM_CD:-}"
```

Comment:
are we going to include this PARAM_CD in the spread.yaml? otherwise it will be allways empty

Reply:
It can be defined in the spread.yaml sure